src/en/administration/federation_getting_started.md

```diff
@@ -6,7 +6,7 @@ Lemmy has three types of federation:
 - BlockList: Explicitly list instances to not connect to. Federation is open to all other instances.
 - Open: Federate with all potential instances.
 
-**Federation is not set up by default.** You can add this [this federation block](https://github.com/lemmynet/lemmy/blob/main/config/config.hjson#L64) to your `lemmy.hjson`, and ask other servers to add you to their allowlist.
+**Federation is enabled by default.** You can add allowed and blocked instances, by adding a comma-delimited list in your instance admin panel. IE to only federate with these instances, add: `enterprise.lemmy.ml,lemmy.ml` to the allowed instances section.
```

Comment:
only real changes are here.

Reply:
To be honest the whole admin section could use much better documentation, to describe what all the options do. Right now there are only input labels and those are not clear unless you are already familiar with Lemmy. It would also be nice to split the settings into categories or tabs like site, federation, rate limit, banned users,